JS/Lesson35/onder/my-app/src/components/Auth/AuthProvider.js

```diff
@@ -0,0 +1,23 @@
+import React, { createContext, useState, useContext } from "react";
+
+const AuthContext = createContext();
+
+export const AuthProvider = ({ children }) => {
+  const [isLoggedIn, setIsLoggedIn] = useState(false);
```

Comment:
Store email and password on the user in the context as well.

Try to use useReducer hook for AuthProvider, it's a better choice as it allow to define explicitly how to handle a certain data change.